using-mobile-extensions/adobe-target/README.md

```diff
@@ -183,64 +182,223 @@ prefetchMboxesList.add(prefetchRequest2);
+```
+
+#### **Example**
+
+```java
+List<String> purchasedProductIds = new ArrayList<String>();
+purchasedProductIds.add("34");
+purchasedProductIds.add("125"); 
+TargetOrder targetOrder = new TargetOrder("123", 567.89, purchasedProductIds);
+
+TargetProduct targetProduct = new TargetProduct("123", "Books");
+
+Map<String, Object> mboxParameters1 = new HashMap<>();
+mboxParameters1.put("status", "platinum");
+
+Map<String, Object> profileParameters1 = new HashMap<>();
+profileParameters1.put("gender", "male");
+
+TargetParameters targetParameters = new TargetParameters.Builder()
+                                    .parameters(mboxParameters1)
+                                    .profileParameters(profileParameters1)
+                                    .product(targetProduct)
+                                    .order(targetOrder)
+                                    .build();
+```
+{% endtab %}
+
+{% tab title="iOS" %}
+#### Syntax
+
+```objectivec
++ (nonnull instancetype) targetOrderWithId: (nonnull NSString*) orderId
+                                     total: (nullable NSNumber*) total
+                       purchasedProductIds: (nullable NSArray <NSString*>*) purchasedProductIds;
+
++ (nonnull instancetype) targetProductWithId: (nonnull NSString*) productId
+                                  categoryId: (nullable NSString*) categoryId;
+
++ (nonnull instancetype) targetParametersWithParameters: (nullable NSDictionary*) targetParameters
+                                      profileParameters: (nullable NSDictionary*) profileParameters
+                                                product: (nullable ACPTargetProduct*) product
+                                                  order: (nullable ACPTargetOrder*) order;
+```
+
+#### Objective-C Example
+
+```objectivec
+NSDictionary *mboxParameters = @{@"status":@"Platinum"};
+NSDictionary *profileParameters = @{@"gender":@"female"};
+
+ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
+
+ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
+
+ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:mboxParameters
+                                                    profileParameters:profileParameters
+                                                              product:product
+                                                                order:order];
+```
+{% endtab %}
+{% endtabs %}
+
+### Using Locations Displayed API for Prefetch
+
+In some situations, you may want to want to inform Target that the corresponding location (mbox) has been viewed. This API sends a display notification to Target for a given prefetched mbox. This helps Target record location display events.
+
+Note: If you're only using regular mboxes, and not prefetching any mbox content, this method should not be called.
```

Comment:
- Change from "Note" to "Tip". 
- Spell out "you're" to "you are"
- Change the end of the sentence to "do not call this method".